AAXtoMP3

```diff
@@ -381,6 +381,9 @@ trap 'rm -r -f "${working_directory}"' EXIT
 # Set up some basic working files ASAP.  Note the trap will clean this up no matter what.
 working_directory=`mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir'`
 metadata_file="${working_directory}/metadata.txt"
+if [[ "${audibleCli}" == "1" && "${mode}" == "single" ]] ; then
```

Comment:
Should I add a comment for this line?

Reply:
Yes, I think it would be nice to shortly explain why we create a new temp file.

Reply:
Okay, done.